ocw/lib/ec2.py

```diff
@@ -352,8 +352,13 @@ def cleanup_images(self, valid_period_days: float) -> None:
             self.log_dbg(f"Found {len(response['Images'])} images in {region}")
             for img in response['Images']:
                 if EC2.is_outdated(parse(img['CreationDate']), valid_period_days):
-                    if self.dry_run:
-                        self.log_info(f"Image deletion {img['ImageId']} skipped due to dry run mode")
+                    tags = img['Tags'] if 'Tags' in img else []
+                    pcw_ignore_tag = next((tag for tag in tags if tag['Key'] == 'pcw_ignore' and tag['Value'] == '1'), None)
```

Comment:
Let's consider any non-zero value as truthy and return False otherwise:

```suggestion
                    pcw_ignore_tag = next((tag for tag in tags if tag['Key'] == 'pcw_ignore' and tag['Value'] != '0'), False)
```

Reply:
Can I do following?
```suggestion
                    pcw_ignore_tag = next((tag for tag in tags if tag['Key'] == Instance.TAG_IGNORE), None)
```
as:
1) I see it in other places in the code
2) I have the impression that that's what @asmorodskyi wants :smile:

Reply:
> Can I do following?
> 
> as:
> 
> 1. I see it in other places in the code

Yes. Much better.